.github/workflows/build.yml

```diff
@@ -26,7 +26,12 @@ jobs:
           username: ${{ secrets.DOCKER_PULL_USERNAME }}
           password: ${{ secrets.DOCKER_PULL_PASSWORD }}
       - name: Build image variant "${{ matrix.variant }}"
-        run: cd ${{ matrix.variant }} && docker buildx build ./ -t ${{ matrix.docker-tag }}
+        run: |
+          docker run --privileged --rm tonistiigi/binfmt --install all
+          docker buildx create --name multibuilder --driver docker-container --bootstrap
+          docker buildx use multibuilder
+          cd ${{ matrix.variant }}
+          docker buildx build --platform linux/arm/v7,linux/arm64/v8,linux/amd64 ./ -t ${{ matrix.docker-tag }}
```

Comment:
You should probably do like https://github.com/sudo-bot/docker-phpmyadmin-snapshots/blob/main/.github/workflows/build.yml

It's less hacks ;)

Reply:
I am afraid I need help from someone who knows Github actions better.

Reply:
I'm a Devops dude, have done gitlab and Jenkins CI / CD, so I shall give it a look. Guess I'll need to fork until I figure it out , otherwise I'll drive everyone batty.

Reply:
@montgomery-auber thanks mate, just make a pull request then please, so I will cancel this messy one.

Reply:
please ping me if you need me to fixup this

Reply:
> please ping me if you need me to fixup this

Yes, I'd be thankful for making these changes, I'm not familiar with Github Actions.

Reply:
I created a new PR that succesfully built and pushed to my Docker hub account.